routes/v1.js

```diff
@@ -22,12 +22,14 @@ var app;
  * GET /siteinfo/{uri}{/prop}
  * Fetches site info for a wiki with the given URI, optionally
  * returning only the specified property. This example shows how to:
- * 1) use named URI parameters (by prefixing them with a double colon)
+ * 1) use named URI parameters (by prefixing them with a colon)
  * 2) use optional URI parameters (by suffixing them with a question mark)
  * 3) extract URI parameters
  * 4) issue external requests
  * 5) use Promises to achieve (4) and return the result
  *
+ * URI parsing doc: https://www.npmjs.com/package/path-to-regexp
```

Comment:
I'd rather recommend [Express' routing guide](http://expressjs.com/guide/routing.html) than this package, as it gives the route context as well and illustrates most of the examples.